### PR TITLE
INT-652 Spam blocking

### PR DESCRIPTION
**Pull request by Onbehalfofme**

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -3653,6 +3704,12 @@ components:
         twilioApiKeySecret:
           type: string
           description: The API Key Secret for the tenants Twilio sub account.
+        blockSpam:
+          type: boolean
+          description: If set to true, tenant turned the spam block on.
+        hangUpSpamCalls:
+          type: boolean
+          description: If set to true, all calls that detected as spam are hanged up immediately.
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:3712]:
```suggestion
          description: This parameter defines if spam blocking is enabled.
        hangUpSpamCalls:
          type: boolean
          description: This parameter defines if the incoming calls that are detected as spam should be hanged up immediately.
```

Comment (BullDog57Rus) [reply talk/talk-api.yaml:3712]: actually, these properties are not needed for that request, I think. So just use these descriptions for `SpamBlockingSettings` and mark them as `required`

**Review by BullDog57Rus — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml [hunk 2]
```diff
@@ -3218,6 +3263,12 @@ components:
           description: If set to true, the call is being recorded now.
         recording:
           $ref: '#/components/schemas/Recording'
+        spam:
+          type: boolean
+          description:This parameter defines if the incoming call is detected as spam.
```
Comment (Onbehalfofme) [inline talk/talk-api.yaml:3268]:
```suggestion
          description: This parameter defines if the incoming call is detected as spam.
```

File: talk/talk-api.yaml [hunk 3]
```diff
@@ -3653,6 +3704,12 @@ components:
         twilioApiKeySecret:
           type: string
           description: The API Key Secret for the tenants Twilio sub account.
+        blockSpam:
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:3707]: maybe we can remove them from this response?

**Review by BullDog57Rus — APPROVED** [review]

**Pull request merged**